packages/fast-tooling-react/src/form/utilities/form.tsx

```diff
@@ -628,3 +628,7 @@ export function getErrorFromDataLocation(
 
     return error;
 }
+
+export function isDefault<T>(value: T | void, defaultValue: T | void): boolean {
+    return typeof value === "undefined" && typeof defaultValue !== "undefined";
```

Comment:
No, because the value could be set and also be equal to the default value.

This checks to see if the value is undefined, and if the default value is not undefined pretty strictly. The other way to do it is `value === undefined` but I prefer using typeof as a legacy check for precision.